Textosaurus no longer builds

Hello, I'm the maintainer of the textosaurus pkgbuilds on the AUR, the compiler used is GCC 11.1.0

#### Brief description of the issue.

Textosaurus no longer builds from the last commit

#### How to reproduce the bug?

```
sudo pacman -Syu git base-devel --needed
git clone https://aur.archlinux.org/textosaurus-git.git
cd textosaurus-git
makepkg -sicC
```

#### What is the expected result?

Textosaurus compile and package successfully

#### What actually happened?

Textosaurus fail to compile

#### Other information (logs, see Wiki)

There are 2 problematic files: 
* `common/miscellaneous/debugging.cpp` 
* `common/dynamic-shortcuts/dynamicshortcuts.cpp`

Here the relevant snippets, attached the there is the complete build() log

```
In file included from common/miscellaneous/debugging.cpp:5:
./definitions/definitions.h:40:49: error: ‘numeric_limits’ is not a member of ‘std’
   40 | constexpr auto MAX_TEXT_FILE_SIZE = qint64(std::numeric_limits<int>::max());
      |                                                 ^~~~~~~~~~~~~~
./definitions/definitions.h:40:43: error: expected primary-expression before ‘(’ token
   40 | constexpr auto MAX_TEXT_FILE_SIZE = qint64(std::numeric_limits<int>::max());
      |                                           ^
./definitions/definitions.h:40:49: error: ‘numeric_limits’ is not a member of ‘std’
   40 | constexpr auto MAX_TEXT_FILE_SIZE = qint64(std::numeric_limits<int>::max());
      |                                                 ^~~~~~~~~~~~~~
./definitions/definitions.h:40:64: error: expected primary-expression before ‘int’
   40 | constexpr auto MAX_TEXT_FILE_SIZE = qint64(std::numeric_limits<int>::max());
      |                                                                ^~~
In file included from ./common/miscellaneous/iofactory.h:6,
                 from common/miscellaneous/iofactory.cpp:3:
./definitions/definitions.h:40:49: error: ‘numeric_limits’ is not a member of ‘std’
   40 | constexpr auto MAX_TEXT_FILE_SIZE = qint64(std::numeric_limits<int>::max());
      |                                                 ^~~~~~~~~~~~~~
./definitions/definitions.h:40:43: error: expected primary-expression before ‘(’ token
   40 | constexpr auto MAX_TEXT_FILE_SIZE = qint64(std::numeric_limits<int>::max());
      |                                           ^
./definitions/definitions.h:40:49: error: ‘numeric_limits’ is not a member of ‘std’
   40 | constexpr auto MAX_TEXT_FILE_SIZE = qint64(std::numeric_limits<int>::max());
      |                                                 ^~~~~~~~~~~~~~
./definitions/definitions.h:40:64: error: expected primary-expression before ‘int’
   40 | constexpr auto MAX_TEXT_FILE_SIZE = qint64(std::numeric_limits<int>::max());
      |                                                                ^~~
```


```
In file included from ./common/dynamic-shortcuts/dynamicshortcuts.h:8,
                 from common/dynamic-shortcuts/dynamicshortcuts.cpp:3:
./definitions/definitions.h:40:49: error: ‘numeric_limits’ is not a member of ‘std’
   40 | constexpr auto MAX_TEXT_FILE_SIZE = qint64(std::numeric_limits<int>::max());
      |                                                 ^~~~~~~~~~~~~~
./definitions/definitions.h:40:43: error: expected primary-expression before ‘(’ token
   40 | constexpr auto MAX_TEXT_FILE_SIZE = qint64(std::numeric_limits<int>::max());
      |                                           ^
./definitions/definitions.h:40:49: error: ‘numeric_limits’ is not a member of ‘std’
   40 | constexpr auto MAX_TEXT_FILE_SIZE = qint64(std::numeric_limits<int>::max());
      |                                                 ^~~~~~~~~~~~~~
./definitions/definitions.h:40:64: error: expected primary-expression before ‘int’
   40 | constexpr auto MAX_TEXT_FILE_SIZE = qint64(std::numeric_limits<int>::max());
      |                                                                ^~~
In file included from ./common/miscellaneous/textfactory.h:6,
                 from common/miscellaneous/textfactory.cpp:3:
./definitions/definitions.h:40:49: error: ‘numeric_limits’ is not a member of ‘std’
   40 | constexpr auto MAX_TEXT_FILE_SIZE = qint64(std::numeric_limits<int>::max());
      |                                                 ^~~~~~~~~~~~~~
./definitions/definitions.h:40:43: error: expected primary-expression before ‘(’ token
   40 | constexpr auto MAX_TEXT_FILE_SIZE = qint64(std::numeric_limits<int>::max());
      |                                           ^
./definitions/definitions.h:40:49: error: ‘numeric_limits’ is not a member of ‘std’
   40 | constexpr auto MAX_TEXT_FILE_SIZE = qint64(std::numeric_limits<int>::max());
      |                                                 ^~~~~~~~~~~~~~
./definitions/definitions.h:40:64: error: expected primary-expression before ‘int’
   40 | constexpr auto MAX_TEXT_FILE_SIZE = qint64(std::numeric_limits<int>::max());
      |                                                                ^~~
./definitions/definitions.h:40:16: warning: ‘MAX_TEXT_FILE_SIZE’ defined but not used [-Wunused-variable]
   40 | constexpr auto MAX_TEXT_FILE_SIZE = qint64(std::numeric_limits<int>::max());
      |                ^~~~~~~~~~~~~~~~~~
```


[textosaurus-git-1:0.9.13.r32.g4e2ad75a-1-x86_64-build.log](https://github.com/martinrotter/textosaurus/files/7833511/textosaurus-git-1.0.9.13.r32.g4e2ad75a-1-x86_64-build.log)

--- a/src/libtextosaurus/definitions/definitions.h
+++ b/src/libtextosaurus/definitions/definitions.h
@@ -3,6 +3,7 @@
 #ifndef DEFINITIONS_H
 #define DEFINITIONS_H
 
+#include <limits>
 #include <QtGlobal>
 
 // *INDENT-OFF*

